server/src/main/java/app/user/UserSearchResponse.java

```diff
@@ -0,0 +1,12 @@
+package app.user;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class UserSearchResponse {
+    private Integer id;
+    private String username;
+    private String imageUrl;
+}

```

Comment:
proszę zobacz też mr poli, napisałem tam konetarz : wszystkie takie klaski powinny w polach mieć informacje o tym czy to pole jest wymagane czy nie . Mianowicie 

public record AuthenticationResponse(
    @Schema(requiredMode = Schema.RequiredMode.REQUIRED) String accessToken,
    @Schema(requiredMode = Schema.RequiredMode.REQUIRED) String refreshToken
) {


to pomaga generatorowi api utworzyc odpowiednie typy